mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -194,6 +194,11 @@
       "run": "after",
       "snippet": "ALTER TYPE job_execution_status ADD VALUE IF NOT EXISTS 'CANCELLED'; ALTER TYPE job_execution_ui_status ADD VALUE IF NOT EXISTS 'CANCELLED';",
       "fromModuleVersion": "mod-source-record-manager-3.4.0"
+    },
+    {
+      "run": "after",
+      "snippet": "DROP TABLE IF EXISTS job_executions CASCADE;",
```

Comment:
Did you check what CASCADE deletes except table?

Reply:
Hi @SerhiiNosko. Cascade deletes only constraints and doesn't affect to any data.